plugins/orders.cpp

```diff
@@ -153,7 +153,7 @@ static void list_library(color_ostream &out) {
         if (name.extension() != ".json")
             continue; // skip non-.json files
         auto sname = name.stem();
-        out << "library/" << sname << std::endl;
+        out << Filesystem::as_string("library" / sname) << std::endl;
```

Comment:
The documented names of the library orders sets use the forward slash. Will this preserve that? Same for stockpiles, blueprint, quickfort, and other tools that come with libraries

Reply:
this only transforms names that are transformed from pathnames enumerated by `listdir_recursive`. input from other sources is completely unaffected